tutorials/examples/orient3d_simulation.py

```diff
@@ -0,0 +1,85 @@
+"""
+This script illustrates the estimation of orientation angles using synchronization
+matrix and voting method, based on simulated data projected from a 3D CryoEM map.
+"""
+
+import os
+import logging
+import numpy as np
+
+import mrcfile
+
+from aspire.source.simulation import Simulation
+
+from aspire.utils.filters import RadialCTFFilter
+from aspire.utils.preprocess import downsample
+from aspire.utils.coor_trans import (register_rotations,
+                                     get_aligned_rotations, get_rots_mse)
+from aspire.orientation.commonline_sync import CommLineSync
+
+logger = logging.getLogger('aspire')
+
+DATA_DIR = os.path.join(os.path.dirname(__file__), '../data/')
+
+logger.info('This script illustrates orientation estimation using '
+            'synchronization matrix and voting method')
+
+# Set the sizes of images 64 x 64
+img_size = 33
+# Set the total number of images generated from the 3D map
+num_imgs = 128
+
+# Set the number of 3D maps
+num_maps = 1
```

Comment:
just curious, has this code been tested with `num_maps = 2`?

Reply:
The orientation estimation algorithm has no option for the number of maps. The num_maps option was set for initializing Simulation oject for Cov3D module but this class was shared for other module. Here we only need num_maps = 1.

Reply:
Yes, I saw that and assumed this was just copy pasta. If I was a user this line would imply this is a setting I would be using, like the ones above it which set the number and size of images.  Maybe can just remove it.

Reply:
We need to keep it to initialize the Simulation object at line 62 but not Orientation object.

Reply:
No we do not need.  It is confusing. You may simply set `C=1`. These "tutorials" are rough enough already, we don't need extra variables floating around because a developer didn't want to remove a line.

Reply:
It depends on how you view the problem. In my view, C=1 is more confusing for beginner. It is hard to tell what is C if we don't explain.

Reply:
You stated we don't need this variable for demonstrating the Orientation Estimation. It is superfluous, and exists because we copy pasted the code from another module (typical).  You are not explaining it in the code here at all, and your out-band explanation in this code review is merely confirming this is copy paste, but calling it "sharing". Give me a break.

Reply:
I am using the same 2D map from the example of cov2d.  I am setting up the simulation object in a very similar way. the num_maps is for simulation object but not directly (indirectly) for orientation estimation. It should be OK. Let me know if this really bother you.

Reply:
This is wasting my time.